dll/win32/msgina/lang/ro-RO.rc

```diff
@@ -58,12 +58,12 @@ BEGIN
     LTEXT "<Message>", IDC_SECURITY_MESSAGE, 15, 62, 245, 18
     LTEXT "<Logon date>", IDC_SECURITY_LOGONDATE, 15, 83, 245, 8
     CTEXT "Ce acțiune preferați?", IDC_STATIC, 10, 108, 255, 8
-    PUSHBUTTON "&Blochează calculatorul", IDC_SECURITY_LOCK, 7, 124, 80, 14
-    PUSHBUTTON "&Deautentifică-mă…", IDC_SECURITY_LOGOFF, 98, 124, 80, 14
-    PUSHBUTTON "Închid&e calculatorul…", IDC_SECURITY_SHUTDOWN, 188, 124, 80, 14
-    PUSHBUTTON "Schim&bă parola…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
-    PUSHBUTTON "&Gestionar de activități", IDC_SECURITY_TASKMGR, 98, 143, 80, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 188, 143, 80, 14
+    PUSHBUTTON "&Blocare calculator", IDC_SECURITY_LOCK, 7, 124, 80, 14
+    PUSHBUTTON "&Deautentificare…", IDC_SECURITY_LOGOFF, 98, 124, 80, 14
+    PUSHBUTTON "Închid&ere…", IDC_SECURITY_SHUTDOWN, 188, 124, 80, 14
+    PUSHBUTTON "Schimba&re parolă…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
+    PUSHBUTTON "&Manager de activități", IDC_SECURITY_TASKMGR, 98, 143, 80, 14
+    PUSHBUTTON "&Revocare", IDCANCEL, 188, 143, 80, 14
```

Comment:
Screenshot of Romanian Windows is required! You are obviously only guessing here!
  PUSHBUTTON "Schimba&re parolă…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
collides with
 PUSHBUTTON "&Revocare", IDCANCEL, 188, 143, 80, 14

Reply:
I wasn't able to find exactly the place where are the words OK and Revocare, but I found they don't have an accelerator. I can send you a screenshot with what I found. These two words don't have any accelerator, so I will remove it everywhere.

![Windows XP Professional-2024-02-26-01-20-25](https://github.com/reactos/reactos/assets/113553683/f77272dd-e1d0-493c-b654-5d66a94b8f4d)

Reply:
Yes, that looks like the best choice then to remove the accels. I do agree.

Reply:
There is a mistake in the Romanian translation that I didn't notice until now. These two words have no accelerator in any file, so every Romanian file must be modified. As you told me, I will try to do less PRs in the future, but with higher quality. It's now that I discover the mistakes.

Reply:
That's a good attitude. The problem with accelerator collisions is, that it does not only render 1 accelerators useless (the one touched), but also it always breaks a 2nd one (the one which it collides).
This conversation is resolved. I just left it uncollapsed because it shows the screenshot.

Reply:
One last question: In your screenshot we can see that &Numere has an accelerator in the source code, but the preview doesn't show it. Maybe you have them configured to not always be shown in your Windows XP!
Since you did not find the exact OK and Revocare buttons (since they might come from the parent control...)

I do recommend you to configure your Windows XP to always show them:
In XP, right-click on the desktop, hit "Properties," head to the Appearance tab, click the "Effects" button, then un-check the "Hide underlined letters ..." option. Now your toolbar menus will always have their keyboard access letters underlined.

Reply:
I'm ready. Please, retest. Please, tell me where is wrong with the word &Numere. I heavily modified rasdlg.dll file in order to look better to XP, so I removed the unnecessary accelerator. Please let me know where is wrong. Maybe it was fixed these days.

Reply:
> One last question: In your screenshot we can see that &Numere has an accelerator in the source code, but the preview doesn't show it. Maybe you have them configured to not always be shown in your Windows XP! Since you did not find the exact OK and Revocare buttons (since they might come from the parent control...)
> 
> I do recommend you to configure your Windows XP to always show them: In XP, right-click on the desktop, hit "Properties," head to the Appearance tab, click the "Effects" button, then un-check the "Hide underlined letters ..." option. Now your toolbar menus will always have their keyboard access letters underlined.

I modified msgina and rasdlg files in order to be both as the XP version ones. Please retest to see if the problem persists.

Reply:
This is for msgina. I send you what you asked me before.

![2024-03-11 15_45_23-](https://github.com/reactos/reactos/assets/113553683/b91caf69-ecf5-46ef-8aa0-1219c8e4a637)

![2024-03-11 15_44_39-Dialog - 1500](https://github.com/reactos/reactos/assets/113553683/2d208223-17bf-4ad9-88a4-97967030b5de)